.bots/header-checker-lint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ignoreFiles": ["protos/*"]
```

Comment:
Are we sure about that?  Instead, I'd rather we make the change in the underlying generator/protos if needed.  If we don't want to block on that, please leave a TODO with a reference to a tracking bug so we don't lose this.

Reply:
@JustinBeckwith created a TODO on this repo, https://github.com/googleapis/nodejs-translate/issues/356

@alexander-fenster could you perhaps create a corresponding tracking issue, for adding license headers, and reference #356.